library.properties

```diff
@@ -0,0 +1,9 @@
+name=MP34DT01
+version=1.0
```

Comment:
Set 1.0.0
Version should be semver compliant even if 1.0 is accepted.
https://github.com/arduino/Arduino/wiki/Arduino-IDE-1.5:-Library-specification#libraryproperties-file-format